nservicebus/hosting/cloud-services-host/hosting-options.md

```diff
@@ -1,84 +0,0 @@
----
```

Comment:
probably not, what would we redirect to? I see no value in that original document anymore and it isn't really redirectable to the cloud services specific content